### ISSUE TITLE
(-8)^(1/3) ≠ -2 and wrong plots for ^(1/3)

### ISSUE THREAD

**Issue by mbartelsm** [serso/android-calculatorpp#201]
Title.
On input ``(-8)^(1/3)`` I get ``1 + 1.7320508076i``

When plotting the function ``x^(1/3)`` I get the following plot:

![Screenshot_2019-12-16-12-12-57-896_org solovyev android calculator](https://user-images.githubusercontent.com/5376208/70902628-8a8e7400-1ffd-11ea-9a2e-d349891ac9b3.jpg)

Which is entirely wrong

**Comment by shadow00** [comment]
Yep, I can replicate this issue on a couple of different devices.

I get a warning that says "Complex number computation in non-RAD mode can lead to unexpected results!".  
If I click "Close" I get 1.9996659589+0.0365520553i (with angles in degrees); if I click "Fix" (thus setting angles in radians) I get the same result as OP.

**Comment by MithicSpirit** [comment]
That is the correct result as it is using the principal root instead of the real-valued root. Would be nice to have an option to prioritize the real-valued root though (also would be a more sensible default).
